tasks2.js

```diff
@@ -0,0 +1,184 @@
+
+// 20. Use trim() to remove any trailing whitespace at the beginning and the end of a string.E.g ' 30 Days Of JavaScript '.
+    console.log(challenge.trim())
+
+// 21. Use startsWith() method with the string 30 Days Of JavaScript and make the result true
+    console.log(challenge.startsWith('30'))
+
+// 22 Use endsWith() method with the string 30 Days Of JavaScript and make the result true
+    console.log(challenge.endsWith('JavaScript'))
+
+// 23 Use match() method to find all the a’s in 30 Days Of JavaScript
+    let pattern = /a/gi
+
+    console.log(challenge.match(pattern))
+
+// 24. Use concat() and merge '30 Days of' and 'JavaScript' to a single string, '30 Days Of JavaScript'
+    let joint = '30 Days of'
+
+    console.log(joint.concat('JavaScript'))
+
+// 25. Use repeat() method to print 30 Days Of JavaScript 2 times
+    console.log(challenge.repeat(2 ));
+
+//  Exercise: Level 2
+
+// 1. Using console.log() print out the following statement: The quote 'There is no exercise better for the heart than reaching down and lifting people up.' by John Holmes teaches us to help one another.
+   let quote = "'There is no exercise better for the heart than reaching down and lifting people up.'\n by John Holmes teaches us to help one another."
+
+   console.log(quote);
+
+   // 2. Using console.log() print out the following quote by Mother Teresa:"Love is not patronizing and charity isn't about pity, it is about love. Charity and love are the same -- with charity you give love, so don't just give money but reach out your hand instead."
+    let quoteMotherTeresa = "\"Love is not patronizing and charity isn\'t about pity, it is about love.\n Charity and love are the same -- with charity you give love, so don\'t just give money but reach out your hand instead.\""
+   
+   console.log(quoteMotherTeresa);
+    // 3. Check if typeof '10' is exactly equal to 10. If not make it exactly equal.
+    let num = '10'
+
+    console.log(num ===10);
+
+    let numInt = +num
+    console.log(numInt);
+
+// 4. Check if parseFloat('9.8') is equal to 10 if not make it exactly equal with 10.
+    let float = '9.8'
+
+    console.log(float===10);
+
+    let numberInt = parseInt(float)
+
+    console.log(numberInt+1);
+
+// 7. Check if 'on' is found in both python and jargon
+    let python = 'python'
+    let jargon = 'jargon'
+
+    console.log(python.includes('on'));
+    console.log(jargon.includes('on'));
+
+// 8. I hope this course is not full of jargon. Check if jargon is in the sentence.
+    let sentenceTwo = 'I hope this course is not full of jargon'
+
+    console.log(sentenceTwo.includes('jargon'));
+
+// 9. Generate a random number between 0 and 100 inclusively.
+    let randomNumber = Math.random(0-100)
+    let numBtnZeroAndHundred = randomNumber*99
+    console.log(numBtnZeroAndHundred);
+
+// 10. Generate a random number between 50 and 100 inclusively.
+function getRandomNumberFifty() {
+    return Math.floor(Math.random() * 101);
```

Comment:
It returns a number between 1 and 100. You need to return a number between 50 and 100.